<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddExistingUser</class>
 <widget class="QDialog" name="AddExistingUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление пользователя</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Фильтр</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>Показать пользователей</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QRadioButton" name="showAllUsers">
             <property name="text">
              <string>Всех</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="showDepartmentsUsers">
             <property name="text">
              <string>По подразделению</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="showFilialsUsers">
             <property name="text">
              <string>По филиалу</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QRadioButton" name="showFilDepUsers">
             <property name="text">
              <string>По филиалу/подразделению</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QRadioButton" name="showFreeUsers">
           <property name="text">
            <string>Не закреплённых ни за одним подразделением</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="margin">
             <number>2</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Филиал</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="filials">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Подразделение</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="departments">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Должность</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="post"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Искать в</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="findIn"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLineEdit" name="searchLine"/>
        </item>
        <item>
         <widget class="QPushButton" name="findButton">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Найти</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/16x16/add/ico/add_filter_16x16.gif</normaloff>:/16x16/add/ico/add_filter_16x16.gif</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearFilterButton">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Снять фильтр</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/16x16/cancelDel/ico/del_filter_16x16.gif</normaloff>:/16x16/cancelDel/ico/del_filter_16x16.gif</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="userView">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Дата прихода на рабочее место:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateInWp">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/16x16/add/ico/add_16x16.png</normaloff>:/16x16/add/ico/add_16x16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Закрыть</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/16x16/cancelDel/ico/button_cancel_16x16 (1).png</normaloff>:/16x16/cancelDel/ico/button_cancel_16x16 (1).png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>showAllUsers</tabstop>
  <tabstop>showFilialsUsers</tabstop>
  <tabstop>showDepartmentsUsers</tabstop>
  <tabstop>showFilDepUsers</tabstop>
  <tabstop>showFreeUsers</tabstop>
  <tabstop>filials</tabstop>
  <tabstop>departments</tabstop>
  <tabstop>post</tabstop>
  <tabstop>findIn</tabstop>
  <tabstop>searchLine</tabstop>
  <tabstop>findButton</tabstop>
  <tabstop>clearFilterButton</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>AddExistingUser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>483</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showAllUsers</sender>
   <signal>clicked(bool)</signal>
   <receiver>filials</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showAllUsers</sender>
   <signal>clicked(bool)</signal>
   <receiver>departments</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showFilialsUsers</sender>
   <signal>clicked(bool)</signal>
   <receiver>filials</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showDepartmentsUsers</sender>
   <signal>clicked(bool)</signal>
   <receiver>departments</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showFilDepUsers</sender>
   <signal>clicked(bool)</signal>
   <receiver>departments</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showFilDepUsers</sender>
   <signal>clicked(bool)</signal>
   <receiver>filials</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showFreeUsers</sender>
   <signal>clicked(bool)</signal>
   <receiver>filials</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showFreeUsers</sender>
   <signal>clicked(bool)</signal>
   <receiver>departments</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showDepartmentsUsers</sender>
   <signal>clicked(bool)</signal>
   <receiver>filials</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showFilialsUsers</sender>
   <signal>clicked(bool)</signal>
   <receiver>departments</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
